core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3027,6 +3034,8 @@ public String toString() {
         "alluxio.master.lineage.recompute.interval";
     public static final String MASTER_LINEAGE_RECOMPUTE_LOG_PATH =
         "alluxio.master.lineage.recompute.log.path";
+    public static final String MASTER_LOG_CONFIG_REPORT_HEARTBEAT_INTERVAL_MS =
+        "alluxio.master.log.config.report.heartbeat.interval.ms";
```

Comment:
no `ms` suffixed for new properties added. `Configuration` now is smart enough to calculate the value in `ms`. thanks to your previous contribution :)